### PR TITLE
Add test case of serializing pandas timestamp

### PR DESCRIPTION
**Pull request by gaugup**
Signed-off-by: Gaurav Gupta <gaugup@microsoft.com>

### CODE REVIEW

File: test/test_serialization_utils.py [hunk 1]
```diff
@@ -58,3 +59,10 @@ def test_serialize_timestamp(self):
         datetime_object = datetime.datetime.strptime(datetime_str, "%Y-%m-%d")
         result = _serialize_json_safe(datetime_object)
         assert datetime_str in result
+
+    def test_serialize_via_json_timestamp(self):
+        timestamp_obj = pd.Timestamp(2020, 1, 1)
+        assert isinstance(timestamp_obj, pd._libs.tslibs.timestamps.Timestamp)
```
Comment (gregorybchris) [inline test/test_serialization_utils.py:65]: Is there a public version of `pd._libs.tslibs.timestamps.Timestamp` we can use?

Comment (imatiach-msft) [reply test/test_serialization_utils.py:65]: maybe pd.tslib.Timestamp?
https://stackoverflow.com/questions/25852044/converting-pandas-tslib-timestamp-to-datetime-python

**Review by imatiach-msft — APPROVED** [review]

File: test/test_serialization_utils.py [hunk 2]
```diff
@@ -58,3 +59,10 @@ def test_serialize_timestamp(self):
...
+        result = _serialize_json_safe(timestamp_obj)
```
Comment (imatiach-msft) [inline test/test_serialization_utils.py:66]: also maybe validate if it is in a list or numpy array if the serialization works?

File: test/test_serialization_utils.py [hunk 3]
```diff
@@ -58,3 +59,10 @@ def test_serialize_timestamp(self):
...
+        assert result is not None
+        assert "2020" in result
```
Comment (imatiach-msft) [inline test/test_serialization_utils.py:68]: shouldn't you also try to json.dumps it, to make sure it is actually json serializable?
_serialize_json_safe just returns the thing in a format that's safe to serialize

File: test/test_serialization_utils.py (hunk 1 above)
Comment (gaugup) [reply test/test_serialization_utils.py:65]: Looks like it is https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Timestamp.html

Comment (gaugup) [reply test/test_serialization_utils.py:65]: Updated.

File: test/test_serialization_utils.py (hunk 3 above)
Comment (gaugup) [reply test/test_serialization_utils.py:68]: Done

File: test/test_serialization_utils.py (hunk 2 above)
Comment (gaugup) [reply test/test_serialization_utils.py]: Done

**Review by gregorybchris — APPROVED** [review]

**Pull request merged**